src/zenml/zen_server/deploy/terraform/terraform_zen_server.py

```diff
@@ -99,6 +99,7 @@ class TerraformServerDeploymentConfig(ServerDeploymentConfig):
         ingress_controller_hostname: The ingress controller hostname to use for
             the ingress self-signed certificate and to compute the ZenML server
             URL.
+        create_sql: Whether to create a SQL database as part of the recipe.
```

Comment:
nit: the name and description are a bit misleading. You're not creating a SQL database, you're deploying a SQL database service. `deploy_db` would be a better name.